include/SFML/Window/Keyboard.hpp

```diff
@@ -154,6 +159,158 @@ class SFML_WINDOW_API Keyboard
+        sLeft,              ///< Keyboard Left Arrow key
+        sDown,              ///< Keyboard Down Arrow key
+        sUp,                ///< Keyboard Up Arrow key
+        sNumLock,           ///< Keypad Num Lock and Clear key
+        sDivide,            ///< Keypad / key
+        sMultiply,          ///< Keypad * key
+        sMinus,             ///< Keypad - key
+        sPlus,              ///< Keypad + key
+        sPadEquals,         ///< keypad = key, probably Mac only
+        sReturn,            ///< Keypad Enter (return) key
+        sDecimal,           ///< Keypad . and Delete key
+        sNumpad1,           ///< Keypad 1 and End key
+        sNumpad2,           ///< Keypad 2 and Down Arrow key
+        sNumpad3,           ///< Keypad 3 and Page Down key
+        sNumpad4,           ///< Keypad 4 and Left Arrow key
+        sNumpad5,           ///< Keypad 5 key
+        sNumpad6,           ///< Keypad 6 and Right Arrow key
+        sNumpad7,           ///< Keypad 7 and Home key
+        sNumpad8,           ///< Keypad 8 and Up Arrow key
+        sNumpad9,           ///< Keypad 9 and Page Up key
+        sNumpad0,           ///< Keypad 0 and Insert key
+        sReverseSolidus,    ///< Keyboard Non-US \ and | key
+        // FIXME what is this one? Might need better name. The doc says:
+        // - Typically near the Left-Shift key in AT-102 implementations.
+        // - Typical language mappings: Belg:<\> FrCa:«°» Dan:<\> Dutch:]|[ Fren:<> Ger:<|> Ital:<> LatAm:<> Nor:<> Span:<> Swed:<|> Swiss:<\> UK:\| Brazil: \|.
+        // What is the difference with "regular" \ and | key?
+        sApplication,       ///< Keyboard Application key
+        sExecute,           ///< Keyboard Execute key
+        sHelp,              ///< Keyboard Help key
+        sMenu,              ///< Keyboard Menu key
+        sSelect,            ///< Keyboard Select key
+        sStop,              ///< Keyboard Stop key
+        sAgain,             ///< Keyboard Again key
+        sUndo,              ///< Keyboard Undo key
+        sCut,               ///< Keyboard Cut key
+        sCopy,              ///< Keyboard Copy key
+        sPaste,             ///< Keyboard Paste key
+        sFind,              ///< Keyboard Find key
+        sMute,              ///< Keyboard Mute key
+        sVolumeUp,          ///< Keyboard Volume Up key
+        sVolumeDown,        ///< Keyboard Volume Down key
+        sLControl,          ///< Keyboard Left Control key
+        sLShift,            ///< Keyboard Left Shift key
+        sLAlt,              ///< Keyboard Left Alt key
+        sLSystem,           ///< Keyboard Left System key
+        sRControl,          ///< Keyboard Right Control key
+        sRShift,            ///< Keyboard Right Shift key
+        sRAlt,              ///< Keyboard Right Alt key
+        sRSystem,           ///< Keyboard Right System key
+
+        sCodeCount          ///< Keep last -- the total number of scancodes
```

Comment:
The s-prefix seems a bit non-SFML like. But I don't have a good alternative to suggest right now.

Reply:
```namespace ScanCodes
{
    enum ScanCodes
    {
        A, B, C, ..
    }
};
```

That's what I do in my Code anyways, if I want implicit conversions to int, which enum class doesn't offer.

Reply:
This is still an issue: the "s" prefix doesn't comply with our coding rules. What about "Scan" as a prefix?

Reply:
@LaurentGomila Does my solution not work? You could do `Scan` for the `namespace` and `Codes` for the `enum`, so when accessing the enum type, you'd do `Scan::Codes` and for individual scancodes, for example `Scan::A`.